inventory/byo/hosts.aep.example

```diff
@@ -27,6 +27,14 @@ deployment_type=atomic-enterprise
 # Enable cluster metrics
 #use_cluster_metrics=true
 
+# In case the metrics setup is in use the url from the setup must be fill in here
```

Comment:
AEP/Origin/OSE have different documentation links so I'd like to make sure we get the right ones. I also want to call out the specific variable name that this relates to. AEP doesn't have it's own docs link yet so we'll use the enterprise link. Could you change this section to

```
# Configure metricsPublicURL in the master config for cluster metrics
# See: https://docs.openshift.com/enterprise/latest/install_config/cluster_metrics.html
#openshift_master_metrics_public_url=https://hawkular-metrics.example.com/hawkular/metrics

# Configure loggingPublicURL in the master config for aggregate logging
# See: https://docs.openshift.com/enterprise/latest/install_config/aggregate_logging.html
#openshift_master_logging_public_url=https://kibana.example.com
```